spec/swagger.yaml

```diff
@@ -5616,6 +5616,53 @@ paths:
                     details: null
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+  /jobs-api/v3/projects/{projectId}/jobs/{translationJobUid}/locales-completion-dates:
+    get:
+      summary: "Get last completion dates per locale of job"
+      tags:
+      - Jobs
+      description: "This get last completion dates per locale of job for all locales that were ever completed."
```

Comment:
i would put description as "Returns the last completion date for any locale completed within the job."